style.css

```diff
@@ -458,3 +471,623 @@ footer div.icons form > button:focus {
+  }
+
+  .about-container .skill-section {
+    padding: 5px 24px 0 24px;
+    display: flex;
+    flex-direction: column;
+    background-image: none;
+    width: 100%;
+    height: 100vh;
+  }
+
+  .about-container div > ul {
+    flex: 1;
+    height: inherit;
+  }
+
+  .about-container div li {
+    width: 100%;
+    text-align: center;
+    list-style: none;
+    margin: 2%;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-size: 19px;
+    font-weight: 400;
+  }
+
+  .about-container div li.frames-title {
+    height: 20px;
+    font-size: 40px;
+    margin-bottom: 25px;
+    font-weight: bold;
+  }
+}
+
+@media (min-width: 768px) {
+  .navbar-container {
+    padding: 0 250px 0 130px;
+    height: 80px;
+    width: 100%;
+    position: fixed;
+  }
+
+  .navbar-container nav {
+    height: inherit;
+  }
+
+  .navbar-container nav ul {
+    display: flex;
+    width: 50%;
+    height: inherit;
+    justify-content: space-between;
+  }
+
+  .navbar-container nav li {
+    font-size: 15px;
+    font-weight: 600;
+    height: 40px;
+    padding: 30px;
+  }
+
+  .navbar-container nav li a {
+    height: inherit;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .navbar-container .logo-item {
+    font-size: 20px;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .navbar-container nav .handberga {
+    display: none;
+  }
+
+  section.intro-container {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 165px 40% 0 3%;
+    gap: 30px;
+    background-image: url("Images/Icons/background.png");
+    background-position: 100%;
+    background-size: 350px;
+    background-position-y: 230px;
+  }
+
+  .intro-text {
+    width: 90%;
+  }
+
+  .intro-icons {
+    width: 10%;
+  }
+
+  section.about-container p {
+    width: 50%;
+    margin-top: 30px;
+    font-size: 16px;
+    line-height: 2rem;
+    margin-left: 90px;
+  }
+
+  section.work-container div p {
+    font-size: 20px;
+    padding-right: 50px;
+  }
+
+  section.about-container h2 {
+    margin-left: 90px;
+  }
+
+  section.intro-container .intro-text h1 {
+    font-size: 57px;
+    width: 100%;
+  }
+
+  section.intro-container .intro-text h2 {
+    font-size: 32px;
+    width: 100%;
+  }
+
+  section.intro-container .intro-text p {
+    font-size: 20px;
+    width: 100%;
+  }
+
+  section.intro-container .intro-icons ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: inherit;
+    width: 100%;
+    gap: 10px;
+  }
+
+  footer div.icons > ul li a img {
+    width: 28px;
+  }
+
+  section.intro-container .intro-icons li {
+    height: inherit;
+    margin-top: 7px;
+    width: 100%;
+  }
+
+  section.intro-container .intro-icons li img {
+    width: 50%;
+  }
+
+  section.recent-work {
+    margin-bottom: 0%;
+  }
+
+  section.about-container div {
+    background-image: url("/Images/abvout.png");
+    background-position: 533px;
+    background-size: 697px;
+    background-position-y: 257px;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, kindly change `background-color` of the `about` section to `#262626` to look like the provided design. :100: 

  - ### Your design
 
   ![image](https://user-images.githubusercontent.com/87197412/186873603-de9d249b-93a9-4b39-bbce-ed7bde431840.png)

   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186873885-79a7b401-ed0f-4a0f-8c47-8e2f4046d951.png)

Reply:
Thanks for the review